StripeFinancialConnections/StripeFinancialConnections/Source/Native/AccountPicker/AccountPickerHelpers.swift

```diff
@@ -89,3 +89,17 @@ extension NSDecimalNumber {
         ]
     }
 }
+
+func buildRetrievingAccountsView() -> UIView {
```

Comment:
this is something I was debating a lot: should I just make a bunch of the supplementary views functions?

for example, instead of:

```
final class MyView {}
```

make it

```
func MyView() -> UIView {}
```

it felt more idiomatic to do the `class` (even if more heavy/'wrong'), but I think I've done both to some extent